x-pack/plugins/beats_management/public/components/table/table_type_configs.tsx

```diff
@@ -83,12 +83,48 @@ export const BeatsTableType: TableType = {
     },
     {
       // TODO: update to use actual metadata field
-      field: 'event_rate',
-      name: 'Event rate',
-      sortable: true,
+      field: 'config_status',
+      name: 'Config Status',
+      render: (value: string, beat: CMPopulatedBeat) => {
+        let color: IconColor = 'success';
+        let statusText = 'OK';
+        let tooltipText = 'Everything looks good!';
+
+        switch (beat.config_status) {
+          case 'REQUIRES_UPDATE':
+            statusText = 'Waiting for Beat checkin';
+            tooltipText = 'Waiting for the Beat to download the latest config';
+          case 'WAITING_FOR_SUCCESS':
+            color = 'warning';
+            statusText = 'Verifying Config';
+            tooltipText = 'Waiting for a status update from the Beat';
+            break;
+          case 'UNKNOWN':
+            color = 'subdued';
+            statusText = 'Offline';
+            if (moment().diff(beat.last_checkin, 'minutes') > 9) {
+              tooltipText = 'This Beat has not connected to kibana in over 10min';
+            }
+            tooltipText = 'This Beat has not yet been started.';
```

Comment:
I think this will always override the previous set?, also, adding the final dot `.` only happens here

Reply:
Ah, good catch, thanks!